### PR TITLE
added post PR reviews

### PR DESCRIPTION
**Pull request by JHalbauer**
Added post PR reviews for `black` and `ruff`.

### CODE REVIEW

**Comment by JHalbauer** [comment]
Error in workflow:
```
Invalid workflow file: .github/workflows/codeql.yml#L46
The workflow is not valid. .github/workflows/codeql.yml (Line: 46, Col: 17): Unexpected value ''
```
@mmacata: "This error has apparently been occurring for months, so it can be ignored in this PR."

**Comment by neteler** [comment]
Seems there is no CodeQL active here? (see also https://github.com/mundialis/github-workflows/pull/29#issuecomment-2476044464)

File: .github/actions/create-upload-suggestions/action.yml [hunk 1]
```diff
@@ -0,0 +1,241 @@
+# from https://github.com/OSGeo/grass/blob/main/.github/actions/create-upload-suggestions/action.yml
```
Comment (mmacata) [inline .github/actions/create-upload-suggestions/action.yml:1]: Are there any changes in this action?
If not, is it possible to remove it here and directly use the action from GRASS GIS in the linting workflow?

**Comment by mmacata** [comment]
> Seems there is no CodeQL active here? (see also [#29 (comment)](https://github.com/mundialis/github-workflows/pull/29#issuecomment-2476044464))

Yes I removed the workflow because it failed for months because there is no language applicable in this repository which can be scanned..

**Comment by neteler** [comment]
You mean in "github-workflows/"? Could a C and a Python dummy file solve it?

**Comment by mmacata** [comment]
Unfortunately not, that was one of the errors from the PR you suggested: https://github.com/mundialis/github-workflows/actions/runs/9971856125/job/27553814828?pr=29

**Comment by mmacata** [comment]
I was reading too fast - dummy files could work! I re-opened your PR from above and for python it works, for C I guess the error appears because my single-line-comment-file cannot be compiled. Let's continue the discussion there as it is off-topic for this PR #29

File: .github/actions/create-upload-suggestions/action.yml (hunk 1 above)
Comment (JHalbauer) [reply .github/actions/create-upload-suggestions/action.yml:1]: No, the files are identical. Using the the action directly from the GRASS repo schould be fine (at least in this version).

**Review by mmacata — APPROVED** [review]
Nice!

**Pull request merged**